streams/src/test/java/org/apache/kafka/test/InternalMockProcessorContext.java

```diff
@@ -57,6 +58,7 @@
     private final Map<String, StateStore> storeMap = new LinkedHashMap<>();
     private final Map<String, StateRestoreCallback> restoreFuncs = new HashMap<>();
     private final ToInternal toInternal = new ToInternal();
+    private final AtomicLong streamTimeSupplier = new AtomicLong();
```

Comment:
`AtomicLong` ?

Reply:
Hm, actually, it's unused. Maybe I'll just throw the "not implemented" exception.